rs/ic_os/network/src/interfaces.rs

```diff
@@ -27,25 +27,21 @@ pub fn has_ipv6_connectivity(
 ) -> Result<bool> {
     // Format with the prefix length
     let ip = format!("{}/{}", generated_ipv6, ipv6_prefix_length);
-    let interface_down_func = || {
-        eprintln!("Removing ip address and bringing interface down");
-        get_command_stdout("ip", ["addr", "del", &ip, "dev", &interface.name])?;
-        deactivate_link(&interface.name)
-    };
 
     eprintln!(
         "Bringing {} up with ip address {}",
         &interface.name,
         &ip.to_string()
     );
+
     get_command_stdout("ip", ["addr", "add", &ip, "dev", &interface.name])?;
     activate_link(&interface.name)?;
 
     let wait_time = Duration::from_secs(2);
     let ping_target = ping_target.parse::<IpAddr>()?;
     let ping_timeout = Duration::from_secs(3);
-    let result = retry(
-        40,
+    let ping_success = retry(
+        10,
```

Comment:
The ping retries was extended from 10 to 40 for this NP support ticket: https://dfinity.atlassian.net/browse/ICSUP-3811

But it ended up not solving the issue. The issue was actually fixed by a firewall reboot.

A shorter timeout means that when we deploy without networking, it won't take an extra minute+ to run through the networking code.